umbraco/Umbraco-CMS
# Initial creation of vertical slice for create member

Initial creation of vertical slice for create member identity user, store and manager. 

Based on the new ASP.NET Core identity implementation for the back office, and [UmbracoIdentity](https://github.com/Shazwazza/UmbracoIdentity).

### Prerequisites

- [x ] I have added steps to test this contribution in the description below

This relates to the larger issue for the whole members migration #9471

### Description

- Initial vertical slice for the back-office **Create Member** functionality (needs fleshing out) to make sure the pattern is correct to follow from
- No database changes
- Started the initial implementation of a store, manager and user for the UmbracoMembers identity implementation
- Started to adjust the controller to use the new manager
- Started with unit tests for new code

To test:

- Run unit tests
- Run integration tests
- Login to back-office and create a member. Member should be persisted to the database as it is currently in Umbraco 8.
- Login to back-office and update a member. Member updates should be persisted to the database as they are in Umbraco 8.
- Login to back-office and delete a member. Member should be deleted from database
